File: profiles/mo6tw/hud/titlemenu.js
root.TitleMenu = {
    Type: TitleMenuType.MO6TW,
    DrawType: DrawComponentType.TitleMenu,
    PressToStartX: 497,
    PressToStartY: 402,
    PressToStartAnimDurationIn: 0.5,
    PressToStartAnimDurationOut: 0.5,
    PressToStartSprite: "TitleMenuPressToStart",
    BackgroundSprite: "TitleMenuBackground",
    CopyrightSprite: "TitleMenuCopyright",
    LogoSprite: "TitleMenuLogo",
    CopyrightX: 465,
    CopyrightY: 662,
    LogoX: 321,
    LogoY: 24,
    MenuBackgroundSprite: "MenuBackground",
    MenuItemLockedSprite: "MenuEntryLocked",
    MenuItemLockedSpriteH: "MenuEntryLockedH",
    MenuEntriesNum: 23,
    MenuEntriesSprites: [],
    MenuEntriesHighlightedSprites: [],
    MenuEntriesX: 513,
    MenuEntriesFirstY: 269,
    MenuEntriesYPadding: 32,
    MenuEntriesTargetWidth: 254,
    PrimaryFadeAnimDuration: 0.3,
    SecondaryMenuAnimTarget: { X: 110, Y: 0 },
    SecondaryMenuPadding: 20,
    SecondaryMenuAnimDuration: 0.512,
    SecondaryMenuAnimUnderX: 0,
    ExtraStoryItemCount: 3,
    ContinueItemCount: 2,
    MemoriesItemCount: 5,
    SystemItemCount: 2,
};

for (var i = 0; i < 23; i++) {
    root.Sprites["MenuEntry" + i] = {
        Sheet: "Title",
        Bounds: {
            X: 641,
            Y: i * 32,
            Width: root.TitleMenu.MenuEntriesTargetWidth,
            Height: 30
        }
    };
    root.TitleMenu.MenuEntriesSprites.push("MenuEntry" + i);
}

for (var i = 0; i < 23; i++) {
    root.Sprites["MenuEntryHighlighted" + i] = {
        Sheet: "Title",
        Bounds: {
            X: 897,
            Y: i * 32,
            Width: root.TitleMenu.MenuEntriesTargetWidth,
            Height: 30
        }
    };
    root.TitleMenu.MenuEntriesHighlightedSprites.push("MenuEntryHighlighted" + i);
}

root.Sprites["MenuEntryLocked"] = {
    Sheet: "Title",
    Bounds: { X: 641, Y: 672, Width: 254, Height: 30 }
};

root.Sprites["MenuEntryLockedH"] = {
    Sheet: "Title",
    Bounds: { X: 897, Y: 672, Width: 254, Height: 30 }
};

root.Sprites["MenuBackground"] = {
    Sheet: "MainBg",
    Bounds: { X: 0, Y: 0, Width: 1280, Height: 720 }
};


root.Sprites["TitleMenuBackground"] = {
    Sheet: "Titlebg",
    Bounds: { X: 0, Y: 0, Width: 1280, Height: 720 },

};

root.Sprites["TitleMenuPressToStart"] = {
    Sheet: "Title",
    Bounds: { X: 257, Y: 193, Width: 302, Height: 30 },
    
};

root.Sprites["TitleMenuCopyright"] = {
    Sheet: "Title",
    Bounds: { X: 1, Y: 225, Width: 384, Height: 30 },
    
};

root.Sprites["TitleMenuLogo"] = {
    Sheet: "Title",
    Bounds: { X: 1, Y: 1, Width: 638, Height: 190 },
   
};

File: profiles/mo8/hud/titlemenu.lua
root.TitleMenu = {
    Type = TitleMenuType.MO8,
    DrawType = DrawComponentType.SystemMenu,
    PressToStartX = 745,
    PressToStartY = 586,
    LogoX = 560,
    LogoY = 24,
    MenuEntriesNum = 13,
    MenuEntriesSprites = {},
    MenuEntriesHighlightedSprites = {},
    MenuEntriesX = 745,
    MenuEntriesFirstY = 586,
    MenuEntriesGalleryFirstY = 546,
    MenuEntriesYPadding = 80,
    NewGameSpriteIndex = 10,
    ContinueSpriteIndex = 9,
    OptionSpriteIndex = 8,
    GallerySpriteIndex = 6,
    AlbumSpriteIndex = 3,
    MusicSpriteIndex = 2,
    ClearListSpriteIndex = 4,
    WarningSpriteIndex = 0,
    AdditionalSpriteIndex = 8,
    DLCSpriteIndex = 1,
    LoadSpriteIndex = 5,
    QuickLoadSpriteIndex = 12,
    HasAdditional = false,
    PressToStartAnimated = false,
    PressToStartAnimDurationIn = 0,
    PressToStartAnimDurationOut = 0,
    PrimaryFadeAnimDuration = 0.5,
    ItemFadeAnimDuration = 0.3,
    PressToStartSprite = "TitleMenuPressToStart",
    BackgroundSprite = "TitleMenuBackground",
    LogoSprite = "TitleMenuLogo"
};

for i = 1, 8 do
    root.Sprites["MenuEntry" .. i] = {
        Sheet = "TitleChip",
        Bounds = {
            X = 0,
            Y = 308 + (i * 88),
            Width = 430,
            Height = 80
        }
    };
    root.TitleMenu.MenuEntriesSprites[#root.TitleMenu.MenuEntriesSprites + 1] = "MenuEntry" .. i;
end

for i = 0, 4 do
    root.Sprites["MenuEntry" .. (i + 10)] = {
        Sheet = "TitleChip",
        Bounds = {
            X = 438,
            Y = 308 + (i * 88),
            Width = 430,
            Height = 80
        }
    };
    root.TitleMenu.MenuEntriesSprites[#root.TitleMenu.MenuEntriesSprites + 1] = "MenuEntry" .. (i + 10);
end

root.TitleMenu.MenuEntriesHighlightedSprites = root.TitleMenu.MenuEntriesSprites;

root.Sprites["TitleMenuPressToStart"] = {
    Sheet = "TitleChip",
    Bounds = { X = 0, Y = 308, Width = 430, Height = 80 },
};

root.Sprites["TitleMenuBackground"] = {
    Sheet = "Title",
    Bounds = { X = 0, Y = 0, Width = 1920, Height = 1080 },
};

root.Sprites["TitleMenuLogo"] = {
    Sheet = "TitleChip",
    Bounds = { X = 0, Y = 0, Width = 810, Height = 300 },
};